app/forms/error_messages.py

```diff
@@ -22,6 +22,7 @@
     "MANDATORY_DATE": lazy_gettext("Enter a date to continue"),
     "MANDATORY_ADDRESS": lazy_gettext("Enter an address to continue"),
     "MANDATORY_DURATION": lazy_gettext("Enter a duration to continue"),
+    "MANDATORY_EMAIL": lazy_gettext("Enter an email address"),
```

Comment:
Did you get confirmation on: https://github.com/ONSdigital/eq-questionnaire-runner/pull/259#discussion_r483037043?

Reply:
Yep, the "to continue" was removed from all error messages a few weeks ago and this got missed as it was still only a prototype at that stage